tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/TelemetryTests.cs

```diff
@@ -342,6 +342,51 @@ public async Task Telemetry_SendsRedactedErrorLogs()
                    .Be("Unable to parse custom sampling rules");
         }
 
+        [SkippableFact]
+        [Trait("Category", "EndToEnd")]
+        [Trait("RunOnWindows", "True")]
+        public async Task Telemetry_CollectsInstallSignature()
+        {
+            var expectedInstallId = "install id";
+            var expectedInstallType = "install type";
+            var expectedInstallTime = "install time";
+
+            EnvironmentHelper.CustomEnvironmentVariables["DD_INSTRUMENTATION_INSTALL_ID"] = expectedInstallId;
+            EnvironmentHelper.CustomEnvironmentVariables["DD_INSTRUMENTATION_INSTALL_TYPE"] = expectedInstallType;
+            EnvironmentHelper.CustomEnvironmentVariables["DD_INSTRUMENTATION_INSTALL_TIME"] = expectedInstallTime;
```

Comment:
Any reason not to use the helpers? 🤔 
```suggestion
            SetEnvironmentVariable("DD_INSTRUMENTATION_INSTALL_ID", expectedInstallId);
            SetEnvironmentVariable("DD_INSTRUMENTATION_INSTALL_TYPE", expectedInstallType);
            SetEnvironmentVariable("DD_INSTRUMENTATION_INSTALL_TIME", expectedInstallTime);
```